pom.xml

```diff
@@ -62,13 +62,11 @@
   </scm>
 
   <properties>
-    <revision>4.3.14</revision>
+    <revision>4.3.17</revision>
     <changelist>-SNAPSHOT</changelist>
-    <gitHubRepo>jenkinsci/google-compute-engine-plugin</gitHubRepo>
-    <jenkins.version>2.346.1</jenkins.version>
+    <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
+    <jenkins.version>2.387.3</jenkins.version>
     <jenkinsRuleTimeout>0</jenkinsRuleTimeout>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
maybe worth to add the minimum jdk requirement in the PR title as is this will be in the release notes

Reply:
> Am I understanding correctly that you are a maintainer of this repository and can therefore edit PR titles?

I''m really sorry for the delay and misunderstanding. But as you are the owner of the current work (this PR) I was under the impression you would have been the most appropriate to modify your own work after review from the member of this project's community. I'm not usually modifying work from others.
But so as it looks to take time, I will do it.